src/components/MovieView/MovieView.jsx

```diff
@@ -1,20 +1,36 @@
 // src/components/MovieView/MovieView.jsx
-import React from "react";
-import PropTypes from "prop-types";
-import Card from 'react-bootstrap/Card';
-import Button from 'react-bootstrap/Button';
-import "./MovieView.scss";
+import React from 'react';
+import { useParams } from 'react-router';
+import PropTypes from 'prop-types';
+import { Card, Button } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import './MovieView.scss';
 
-export const MovieView = ({ movie, onBackClick }) => {
+export const MovieView = ({ movie, addFavorite, removeFavorite, isFavorite }) => {
```

Comment:
In your Router in the `MainView` component you have passed the `movies` props and NOT `movie` to the `MovieView` component. You should also import `movies` and NOT `movie` as you have done here.